ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -4,63 +4,60 @@
     cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
   run_once: true
 
-- oc_obj:
+- name: "Get the host's pod network subnet"
+  oc_obj:
     state: list
     kind: hostsubnet
   register: hostsubnets
   delegate_to: "{{ cluster_master }}"
   run_once: true
 
+- name: "Get the cluster's network config"
+  oc_obj:
+    state: list
+    kind: clusternetworks
+    name: default
+  register: clusternetworks
+  delegate_to: "{{ cluster_master }}"
+  run_once: true
+
 - set_fact:
     kubernetes_node_name: "{{ (hostvars[inventory_hostname].ec2_private_dns_name if hostvars[inventory_hostname].oo_location == 'aws' else inventory_hostname) }}"
+    servicenetwork: "{{clusternetworks.results.results[0].serviceNetwork}}"
 
-- set_fact:
     node_cidr: |-
       {%- for item in hostsubnets['results']['results'][0]['items'] -%}
        {%- if item.host == kubernetes_node_name -%}
         {{ item.subnet }}
        {%- endif -%}
       {%- endfor -%}
 
-- set_fact:
     oo_udp_chain:
     - chain: oo-udp
-      after: "filter$"
+      after: "^\\*filter$"
       regex: :oo-udp - \[0:0\]
       block:  ":oo-udp - [0:0]"
 
     oo_udp_limit_chain:
     - chain: oo-udp-limit
-      after: "filter$"
+      after: "^\\*filter$"
       regex: :oo-udp-limit - \[0:0\]
       block:  ":oo-udp-limit - [0:0]"
 
     filtering_udp_rules:
-    # allow all outbound UDP for privileged pods
-    # send all other traffic to the oo-udp chain so that rules that allow
-    # traffic can call -j RETURN instead of -j ACCEPT, and thus permit us
-    # to do later accounting
+    # Send network-namespaced pod traffic to the oo-udp chain. 
     - chain: OPENSHIFT-OUTPUT-FILTERING
       after: "OPENSHIFT-OUTPUT-FILTERING"
-      regex: "id_outbound_udp_filtering_2_"
-      line: "-I OPENSHIFT-OUTPUT-FILTERING -m comment --comment id_outbound_udp_filtering_2_ -s {{ node_cidr }} -o eth0 -p udp -j oo-udp"
```

Comment:
looks like you got rid of the wrong one. We want to keep the -j oo-udp rule and lose the -j ACCEPT rule. Sorry if I made that confusing.